src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/PrisonApiService.kt

```diff
@@ -45,4 +51,24 @@ class PrisonApiService(
 
     return null
   }
+
+  fun getPrisonerHousingLocation(prisonerId: String): PrisonerHousingLocationsDto? {
+    try {
+      return prisonApiClient.getPrisonerHousingLocation(prisonerId)
+    } catch (e: WebClientResponseException) {
+      if (e.statusCode != HttpStatus.NOT_FOUND) {
+        LOG.error("Exception thrown on prison API call - /api/offenders/$prisonerId/housing-location", e)
+        throw e
+      }
+    }
+
+    return null
+  }
+
+  fun getPrisonerDetails(prisonerId: String): PrisonerDetailsDto? {
+    return if (prisonApiLevelsEndpoint == PrisonApiGetLevelsEndpoint.FULL_STATUS)
+      getPrisonerFullStatus(prisonerId)
```

Comment:
Not sure we should be doing this, once new api is working remove old methods..

Reply:
Updated.